Njord/src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -23,15 +38,56 @@ public class ElevatorSubsystem extends SubsystemBase {
   public enum ElevatorState {
     S_GoingUp, S_GoingDown, S_Stopped
     }
+
+  private final MutVoltage mVoltage = Volts.mutable(0);
+  private final MutDistance mDistance = Meters.mutable(0);
+  private final MutLinearVelocity mVelocity = MetersPerSecond.mutable(0);
+
+private Voltage setVolts = (Voltage volts) -> setVoltage(volts);
+private double setLog = (SysIdRoutineLog) -> setLog(SysIdRoutineLog);
+
+
+
+private final SysIdRoutine mSysIdRoutine = 
+   new SysIdRoutine(
+     new SysIdRoutine.Config(Volts.per(Seconds).of(1.2), Volts.of(12), Seconds.of(10)),
+     new SysIdRoutine.Mechanism(
+      voltage -> {
+        elevatorMotor1.setVoltage(voltage);
+      },
+        log -> {
+        log.motor("drive")
+        .voltage(
+         mVoltage.mut_replace(
+           elevatorMotor1.get() * RobotController.getBatteryVoltage(), Volts)
+          .linearPosition(mDistance.mut_replace(elevatorMotor1.getPosition().getValueAsDouble(), Meters)
```

Comment:
I don't think this is meters. Meters would make sense for driving. What other units can be used?

Reply:
Instead of `linearPosition` motor logging has `angularPosition` which will work much better for the elevator (I hope).

```suggestion
          .angularPosition(elevatorMotor1.getPosition().getValue())
```

Again this means that `mDistance` is not needed because a unit conversion is not needed.